CHANGELOG.md

```diff
@@ -25,27 +25,38 @@ We'd like to offer a big thanks to the 11 contributors who made this release pos
 
 - 📚 Documentation improvements
 
-### `@mui/x-data-grid@v6.9.2` / `@mui/x-data-grid-pro@v6.9.2` / `@mui/x-data-grid-premium@v6.9.2`
+### Data Grid
 
-#### Changes
+#### `@mui/x-data-grid@6.9.2`
 
 - [DataGrid] Fix `RangeError` when using flex columns (#9554) @cherniavskii
 - [DataGrid] Fix React 17 editing bug (#9530) @romgrk
 - [DataGrid] Use `getRowId` in filtering (#9564) @romgrk
 - [DataGrid] Correctly reflect `TablePagination`'s `rowsPerPageOptions` shape to `pageSizeOptions` (#9438) @burakkgunduzz
-- [DataGridPremium] Auto-scroll when making range selection (#8661) @m4theushw
 - [l10n] Improve Spanish (es-ES) locale (#9500) @fufex
 
-### `@mui/x-date-pickers@v6.9.2` / `@mui/x-date-pickers-pro@v6.9.2`
+#### `@mui/x-data-grid-pro@6.9.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link)
```

Comment:
Where is this file used in our webpage such that the defined redirect would take effect? 🤔 
[Opening in GitHub](https://github.com/mui/mui-x/blob/master/CHANGELOG.md) shows unloadable images, which don't look that great. 🤔

Reply:
These redirects will start working after we deploy our docs to production.
I'll cherry-pick and deploy the redirects changes to fix this

Reply:
Sorry for the noob question. Thanks, @cherniavskii for handling it. 👍

Reply:
Thanks for pointing this out! 🙂

Reply:
@cherniavskii thanks for the cherry-pick, I thought these broken for a week, until the next release wouldn't be too bad 😁.